### PR TITLE
Release 0.12.0

### PR DESCRIPTION
**Pull request by phoracek**
- Added OVS CNI component
- Bumped KubeMacPool version to v0.4.0
- Fixed RBAC for OKD4

### CODE REVIEW

File: manifests/cluster-network-addons/0.12.0/network-addons-config-example.cr.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  imagePullPolicy: IfNotPresent
+  kubeMacPool: {}
```
Comment (qinqon) [inline manifests/cluster-network-addons/0.12.0/network-addons-config-example.cr.yaml]: "kubeMacPool": {
              "rangeStart": "02:00:00:00:00:00",
              "rangeEnd": "FD:FF:FF:FF:FF:FF"
            },

Comment (phoracek) [reply manifests/cluster-network-addons/0.12.0/network-addons-config-example.cr.yaml]: It is omitted on purpose. By default user doesn't care about the range, they care that there are no collisions. If they want to have a specific range, they can follow README

Comment (qinqon) [reply manifests/cluster-network-addons/0.12.0/network-addons-config-example.cr.yaml]: Fair enough

**Comment by phoracek** [comment]
Looks like we don't need it, HCO started working. We can postpone release for now.

**Comment by SchSeba** [comment]
lets try to run it again

**Comment by SchSeba** [comment]
ci test please

**Comment by phoracek** [comment]
We added more components to the operator and therefore it takes more time to start the whole thing. Let me increase our timeouts.

**Pull request closed without merging**